### PR TITLE
Resource tests

### PR DESCRIPTION
**Pull request by miah**
Tests for Chef::Resource::Java

### CODE REVIEW

File: test/spec/resource_java_spec.rb [hunk 1]
```diff
@@ -1,11 +1,49 @@
-require 'minitest/autorun'
+require_relative 'spec_helper'
+require 'resource_java'
 
 describe 'Chef::Resource::Java' do
-  describe 'install_options' do
+  let(:node) do
```
Comment (whilp) [inline test/spec/resource_java_spec.rb:5]: I've looked before, but is there a convenient way to cook this boilerplate setup into a cookbook-global rspec configuration?

**Comment by whilp** [comment]
+1

File: test/spec/resource_java_spec.rb (hunk 1 above)
Comment (miah) [inline test/spec/resource_java_spec.rb:5]: Probably. I haven't really looked to deeply into writing such things in minitest. Its definitely a large body of code that is mostly identical across chef resources. I could probably write something to automate its inclusion on resources (and similarly on providers).

Comment (whilp) [inline test/spec/resource_java_spec.rb:5]: Ah, I meant an rspec config in spec_helper.rb that would set up this node context for all similar specs.

Comment (miah) [inline test/spec/resource_java_spec.rb:5]: Ya, I know nothing about rspec. Never used it =)

**Pull request merged**